local-dev/.gitignore

```diff
@@ -0,0 +1,9 @@
+# Ignore everything in this directory...
+*
+./terra-helmfile/**
+
+# Except...
```

Comment:
Is there a subdirectory we can use instead of needing to add exceptions for everything in this directory? This is going to be a little surprising when we try to add something new.

Reply:
Sure, done! (Fwiw I was following the existing pattern in WSM and Buffer.)